canvas_modules/common-canvas/src/common-properties/components/moveable-table-rows/moveable-table-rows.jsx

```diff
@@ -230,5 +244,13 @@ MoveableTableRows.propTypes = {
 	setCurrentControlValueSelected: PropTypes.func.isRequired,
 	setScrollToRow: PropTypes.func.isRequired,
 	tableContainer: PropTypes.object.isRequired,
-	disabled: PropTypes.bool
+	disabled: PropTypes.bool,
+	disableRowMoveButtonsForPropertyIds: PropTypes.array // set by redux
 };
+
+const mapStateToProps = (state, ownProps) => ({
+	// Array of propertyIds - propertyId used for disabling row move buttons
+	disableRowMoveButtonsForPropertyIds: state.disableRowMoveButtonsReducer.propertyIdArray
```

Comment:
It would be better to have a method in the propertyController to get the state for a specific property.  This saves on rerendering the react component.  Having the entire array means each change to that array rerender every component listening to that array.